_sass/modules/_button.scss

```diff
@@ -17,6 +18,15 @@ $button-font-size: 1.1rem;
 
 .button--ac {
   @include button-color($ac-brand, $ac-brand, $white, $ac-brand);
+
+  &[data-current] {
```

Comment:
@informatykgosia I'm not sure now if you had such cases (but I think it would be the same at least for the top nav) but you could just use `[data-current]` without specifics if you did :)